src/payment/instrument/supported-payment-instruments.ts

```diff
@@ -49,7 +49,7 @@ const supportedInstruments: SupportedInstruments = {
         provider: 'paymetric',
         method: 'credit_card',
     },
-    'barclaycard.card': {
+    'barclaycard.credit_card': {
```

Comment:
Why does 'barclaycard.credit_card' include the method type when all others are only the provider?

Also, will there be any use cases of  'barclaycard.card' which will subsequently stop working after this change?

Reply:
Barclaycard is a provider with multiple APMs (Alternate Payment Methods), given this scenario, the backend always maps the provider as the `gateway` for every method, then when the `gateway` field is present the sdk concatenates `gateway +  method` to get the key that is used to search in this file.

file `src/payment/instrument/instrument-selector.ts`:
``` ts
    const paymentMethodKey = paymentMethod.gateway ? `${paymentMethod.gateway}.${paymentMethod.id}` : paymentMethod.id;
    const currentMethod = supportedInstruments[paymentMethodKey];
```

The backend changes needed to rename `card` to `credit_card` in the backend have already been merged, so the cases where 'barclaycard.card' is used will no longer be happening.